### ISSUE TITLE
using "login user" for ssh connection when "system user" is empty

### ISSUE THREAD

**Issue by jonathan-orsel** [skavanagh/KeyBox#222]
Hello,
Thank you for your share, keybox is a very nice project!

For compliance reason we have to use personal account on every linux ssh access, no way to change it unfortunately!
We are about 20 administrators and we manage 150+ servers.

Having a functionality such as discussed in https://github.com/skavanagh/KeyBox/issues/122 is for us a necessity.
For example if the "system user" is empty or is set to "$login" the ssh session would be created using the login user name.
My java skills are quite limited but I pretty sure that a minor update in manage/util/SSHUtil.java at the  jsch.getSession(...) lines would do the trick.
Do you think you will add such a functionality soon? I also could pull a contribution if you prefer but my coding is a bit ugly :-)

**Comment by chrisarceo26** [comment]
Yep thats also my concern. If that would be implemented, the application will be perfect and i guess the best of all ssh access manager regards its main function!

**Comment by ffabreti** [comment]
+1
Bastillion could `adduser` on every remote system on user profile association and `userdel` on user/profile removal
